components/2024/layout/navbar.jsx

```diff
@@ -1,67 +1,75 @@
 "use client";
-
-import Image from "next/image";
-import Link from "next/link";
-import { QUICKET_LINK } from "@/2023/utils/constants";
-import { useState } from "react";
+import { useState, Fragment } from "react";
+import { Dialog, Disclosure, Popover, Transition } from "@headlessui/react";
```

Comment:
hey @GeoXhacker , the redesign looks amazing...i really love it.

`headlessui` is great, its actually what we'd piloted with but, due to a couple of reasons not limited to the numbers of components they offer.... we were moving to [radixui](https://www.radix-ui.com/) implemented with [shadcn](https://ui.shadcn.com/) and [lucid icons](https://lucide.dev/icons/) .

it would really be amazing to have this navbar with the new approach we adopting moving forward. I understand [shadcn](https://ui.shadcn.com/) requires typescript, but, we have a work around that in progress.  you can please check out what @amkayondo is working on via #131 

it would actually save us a bunch of dev hours if we have this navbar implementation off a fork of #131 so that, we dont move in circles.

really excited to have the implementation here to see us have this redesign! Thanks a bunch, @GeoXhacker!